dask/base.py

```diff
@@ -23,6 +23,41 @@ class Base(object):
 
     def visualize(self, filename='mydask', format=None, optimize_graph=False,
                   **kwargs):
+        """
+        Render the computation of this object's task graph using graphviz.
+
+        Requires `graphviz` to be installed.
```

Comment:
The way I understood it was single backticks are only good when referencing variables to the function by their name, everything else should be double backticks. Either way, sphinx often doesn't do code-highlighting for things in single backticks, and we end up with odd italics everywhere instead. This may be an issue with our sphinx config though.